pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -312,6 +311,38 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
+func getDruidDeployCommands(gardenletConf *config.GardenletConfiguration) []string {
+	command := []string{"" + "/bin/etcd-druid"}
+	command = append(command, "--enable-leader-election=true")
+	command = append(command, "--ignore-operation-annotation=false")
+	if gardenletConf == nil {
+		return command
+	}
+
+	config := gardenletConf.ETCDConfig
+	if config == nil {
+		return nil
```

Comment:
Isn't it ok to return `command` here rather than `nil`? Basically, if `etcdConfig` is nil, we can simply return the command with default values like `--workers=50,--custodian-workers=10` to maintain backward compatibility. WDYT?

Reply:
Yes. Bad mistake. Thanks

Reply:
I returned the command only. It will pick diffault values if the configs are not given

Reply:
> It will pick diffault values if the configs are not given

Can you explain? Image one upgrades Gardener and does not configure the new settings in the Gardenlet-Componentconfig, i.e. `gardenletConf.ETCDConfig == nil`. I'd expect that we then still use the "opinionated" configuration that we used earlier (see [here](https://github.com/gardener/gardener/blob/14c807d460bc27176b0d3d0ff9fd6c11148231f4/pkg/operation/botanist/component/etcd/bootstrap.go#L244-L247)) (-> `--workers=50` is missing).

Reply:
Hmm.. Then we should only add `--workers=50` for backward compatibility. For custodian workers, it was anyway picking default value

Reply:
Yes, fine for me in this case 👍